external/storm-kafka-client/src/main/java/org/apache/storm/kafka/spout/StormStringDeserializer.java

```diff
@@ -15,22 +15,11 @@
  *   See the License for the specific language governing permissions and
  *   limitations under the License.
  */
-
 package org.apache.storm.kafka.spout;
 
-import org.apache.kafka.clients.consumer.ConsumerRecord;
-
-import java.util.List;
-
-public class KafkaSpoutTuplesBuilderWildcardTopics<K,V> implements KafkaSpoutTuplesBuilder<K,V> {
-    private KafkaSpoutTupleBuilder<K, V> tupleBuilder;
+import org.apache.kafka.common.serialization.StringDeserializer;
 
-    public KafkaSpoutTuplesBuilderWildcardTopics(KafkaSpoutTupleBuilder<K, V> tupleBuilder) {
-        this.tupleBuilder = tupleBuilder;
-    }
+public class StormStringDeserializer extends StringDeserializer implements SerializableDeserializer<String> {
```

Comment:
Do we still need this? users can configure the StringSerializer from kafka-clients lib